### DISCUSSION TITLE
How to build and watch without starting the app?

### DISCUSSION THREAD

**Discussion by merwaaan** [alex8088/electron-vite#232] [Q&A]
Is there a known way to rebuild the app when its source code changes **without having the electron app start**?

From what I tried:
- `electron-vite` starts the app and watches for changes
- `electron-vite build` and `electron-vite preview` build the app just once

Something like `electron-vite build --watch` would sound perfect but it's not supported by the CLI and hacking electron-vite to inject a --watch flag anyway does not seem to do anything.

Basically, I'm interested in having immediate feedback and short iterative builds when working, but I don't need to have the application starting because it uses some features that do no work well with hot reload anyway (eg web workers)

Thanks for your help!

**Comment by alex8088** [comment]
- Hot reloading can only be used during the development. https://electron-vite.org/guide/hot-reloading.html

Maybe  `--rendererOnly` arg meet you

The `--rendererOnly` option is only used for dev command to skip the main process and preload scripts builds, and start dev server for renderer only. This option will greatly increase dev command speed.
https://electron-vite.org/guide/cli.html#options

**Comment by merwaaan** [comment]
Thanks @alex8088 

I'm not interested in hot-reloading, actually i want to disable it explictly.

Ideally, I would just want the iterative builds to not start the app.
